tamagotchi-01/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+#![no_std]
+use gstd::{msg, prelude::*, exec};
+
+use interface::{Tamagotchi,Action,Event};
```

Comment:
Please use `cargo fmt` before committing. 
```suggestion
use interface::{Tamagotchi, Action, Event};
```